src/CalcViewModel/DataLoaders/UnitConverterDataLoader.h

```diff
@@ -87,7 +87,7 @@ namespace CalculatorApp
             std::wstring GetLocalizedStringName(_In_ Platform::String ^ stringId);
 
             std::shared_ptr<std::vector<UnitConversionManager::Category>> m_categoryList;
-            std::shared_ptr<UnitConversionManager::CategoryToUnitVectorMap> m_categoryToUnits;
+            std::shared_ptr<UnitConversionManager::CategoryToUnitVectorMap> m_categoryIDToUnitCollectionMap;
```

Comment:
nit: I think "Collection" is redundant with "Map"--maybe "m_categoryIDToUnitMap" is a better name?

Reply:
was thinking of a way to name it to convey that units is actually a collection of units not a singular, i'll just name it m_categoryIDToUnitsMap